commission/reports/report_settlement_templates.xml

```diff
@@ -9,7 +9,7 @@
                         <address
                             class="mb-0"
                             t-field="o.agent_id"
-                            t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}'
+                            t-options="{'widget': 'contact', 'fields': ['address', 'name'], 'no_marker': True}"
```

Comment:
Why changing this?

Reply:
you're right, my VScode change this pattern inappropriately